packages/@aws-cdk/app-delivery/lib/pipeline-deploy-stack-action.ts

```diff
@@ -97,7 +97,7 @@ export class PipelineDeployStackAction extends cdk.Construct {
   /**
    * The role used by CloudFormation for the deploy action
    */
-  public readonly role: iam.Role;
+  public readonly role: iam.IRole;
```

Comment:
This *is* a breaking change (code that expects to find `Role` here now finds something more abstract).